documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc

```diff
@@ -171,167 +172,14 @@ You can change the frequency by adding `refresh.topics.interval.seconds` to the
-Offsets for the `checkpoints` topic are tracked at predetermined intervals through configuration.
-Both topics enable replication to be fully restored from the correct offset position on failover.
-
-The location of the `offset-syncs` topic is the `source` cluster by default.
-You can use the `offset-syncs.topic.location` connector configuration to change this to the `target` cluster.
-You need read/write access to the cluster that contains the topic.
-Using the target cluster as the location of the `offset-syncs` topic allows you to use MirrorMaker 2 even if you have only read access to the source cluster.
-
-== Synchronizing consumer group offsets
-
-The `__consumer_offsets` topic stores information on committed offsets for each consumer group.
-Offset synchronization periodically transfers the consumer offsets for the consumer groups of a source cluster into the consumer offsets topic of a target cluster.
-
-Offset synchronization is particularly useful in an _active/passive_ configuration.
-If the active cluster goes down, consumer applications can switch to the passive (standby) cluster and pick up from the last transferred offset position.
-
-To use topic offset synchronization, enable the synchronization by adding `sync.group.offsets.enabled` to the checkpoint connector configuration, and setting the property to `true`.
-Synchronization is disabled by default.
-
-When using the `IdentityReplicationPolicy` in the source connector, it also has to be configured in the checkpoint connector configuration.
-This ensures that the mirrored consumer offsets will be applied for the correct topics.
-
-Consumer offsets are only synchronized for consumer groups that are not active in the target cluster.
-If the consumer groups are in the target cluster, the synchronization cannot be performed and an `UNKNOWN_MEMBER_ID` error is returned. 
-
-If enabled, the synchronization of offsets from the source cluster is made periodically.
-You can change the frequency by adding `sync.group.offsets.interval.seconds` and `emit.checkpoints.interval.seconds` to the checkpoint connector configuration.
-The properties specify the frequency in seconds that the consumer group offsets are synchronized, and the frequency of checkpoints emitted for offset tracking.
-The default for both properties is 60 seconds.
-You can also change the frequency of checks for new consumer groups using the `refresh.groups.interval.seconds` property, which is performed every 10 minutes by default.
-
-Because the synchronization is time-based, any switchover by consumers to a passive cluster will likely result in some duplication of messages.
-
-NOTE: If you have an application written in Java, you can use the `RemoteClusterUtils.java` utility to synchronize offsets through the application. The utility fetches remote offsets for a consumer group from the `checkpoints` topic. 
-
-
-== Deciding when to use the heartbeat connector
```

Comment:
Did this content move somewhere else? I didn't spot a replacement section about using the `MirrorHeartbeatConnector`

Reply:
Thanks @katheris. I reduced it down to the following in the intro to this section:

_(Optional) Monitors connectivity between clusters. Useful for alerts but not required for replication._ 

I'm trying to trim to only what's necessary, but I can expand more if there is anything in this section we should retain about the heartbeat connector?

Reply:
Hi @katheris -- I've introduced a new section for the heartbeat connector in the last commit